server/src/main/java/org/activityinfo/server/query/JsonResponseBuilder.java

```diff
@@ -0,0 +1,117 @@
+package org.activityinfo.server.query;
+
+import com.google.common.base.Preconditions;
+import com.google.gson.stream.JsonWriter;
+import org.activityinfo.model.query.ColumnSet;
+import org.activityinfo.model.query.ColumnView;
+import org.activityinfo.model.query.QueryModel;
+import org.activityinfo.model.resource.ResourceId;
+import org.activityinfo.store.query.impl.ColumnSetBuilder;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.StreamingOutput;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Builds a JSON response
+ */
+public class JsonResponseBuilder {
+
+    private ColumnSetBuilder builder;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "builder" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)